packages/@sfx/core/test/unit/common/utils/core.test.ts

```diff
@@ -0,0 +1,315 @@
+import { expect, spy } from '../../../utils';
+import {
+  calculateMissingDependencies,
+  initPlugins,
+  readyPlugins,
+  registerPlugins,
+} from '../../../../src/utils/core';
+
+describe('CoreUtils', () => {
+  describe('calculateMissingDependencies()', () => {
+    it('should return missing dependencies', () => {
+      const registry = {
+        a: 'aa',
+        b: 'bb',
+        c: 'cc',
+      };
+      const plugins: any = [
+        {
+          metadata: {
+            name: 'm',
+            depends: ['a', 'x'],
+          },
+        },
+        {
+          metadata: {
+            name: 'n',
+            depends: ['z'],
+          },
+        },
+      ]
+
+      const missing = calculateMissingDependencies(plugins, registry);
+
```

Comment:
You can remove this newline, and consider doing away with the one-use variable.

Reply:
The idea with having this newline and the one-use variable is to split the test into three parts: [Arrange, Act, and Assert](http://wiki.c2.com/?ArrangeActAssert). I would rather optimize for clarity than performance in the tests.

Reply:
Acknowledged. I'll keep it in the spirit of having more consistent-looking tests across the project and the org.